service/dap/server.go

```diff
@@ -831,83 +832,79 @@ func (s *Session) onLaunchRequest(request *dap.LaunchRequest) {
 			FailedToLaunch, "Failed to launch", fmt.Sprintf("invalid debug configuration - %v", err))
 		return
 	}
+	s.config.log.Debug("parsed launch config: ", prettyPrint(args))
```

Comment:
Is there any reason to do this before setting args.Mode to debug below?

Reply:
My goal was to log what the user supplied and how we interpreted it. It's not exact because a couple of defaults are populated before the parsing, but this on is not one of them (and I cannot remember why.)